OneCommunityGlobal/HighestGoodNetworkApp
# Tapan Fixed the Pagination UI in Members Table for Project Reports

# Description
<img width="752" alt="Screenshot 2024-04-26 at 4 45 30 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73526712/526d0566-5323-4505-8c5c-5c19b80bf0dc">

Pagination Overlaps with table values. 

## Related PRS (if any):
N/A
…

## Main changes explained:
- Fixed UI for handling overlap of pagination and table values
- Fixed CSS styling for responsiveness 

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin/owner user
5. Go to Reports -> Projects -> Any project with 7 or more members
6. Make Screen Smaller
7. Check that pagination doesn't overlaps with table values

## Screenshots or videos of changes:

Before - 

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73526712/15e5ec5c-235b-4ece-b1ed-0bbaa775f5b8


After - 


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73526712/f67c176e-af26-4a60-873b-c95ffea8a76d
